### PR TITLE
Implement Filesystem pages

### PR DESCRIPTION
**Pull request by ip1981**
Ref. #1354 

fs

fs1

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1497)

### CODE REVIEW

File: iml-gui/crate/src/components/pie_chart.rs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+// license that can be found in the LICENSE file.
+
+use seed::{attrs, circle, path, prelude::*, style, svg};
+use std::f64::consts::PI;
+
+pub(crate) fn pie_chart<T>(ratio: f64) -> Node<T> {
+    let radians = 2.0 * PI * ratio;
+    let end_x = radians.cos();
+    let end_y = radians.sin();
+    let large_arc_flag = if ratio > 0.5 { 1 } else { 0 }; // FIXME: negative ratio?
```
Comment (johnsonw) [inline iml-gui/crate/src/components/pie_chart.rs]: Couldn't this also be:
```
let large_arc_flag = ratio.round()
```

File: iml-gui/crate/src/page/filesystem.rs [hunk 2]
```diff
@@ -1,10 +1,229 @@
...
+        server_link(Some(&t.primary_server), &t.primary_server_name)
+    } else {
+        plain!("N/A")
+    }
+}
+
+pub(crate) fn clients_view<I>(f: &Filesystem) -> Node<I> {
+    plain!(match f.client_count {
+        Some(c) => c.round().to_string(),
+        None => "N/A".to_string(),
```
Comment (johnsonw) [inline iml-gui/crate/src/page/filesystem.rs]: What's the difference between `plain!("N/A")` and `"N/A".to_string()`?

Comment (jgrund) [reply iml-gui/crate/src/page/filesystem.rs]: huh, looks like an alias for a new text node, neat.

File: iml-gui/crate/src/components/pie_chart.rs (hunk 1 above)
Comment (ip1981) [reply iml-gui/crate/src/components/pie_chart.rs]: Yeah, it looks like round() :)

**Pull request merged**